ludwig/decoders/llm_decoders.py

```diff
@@ -110,23 +110,31 @@ def get_prediction_set(self):
 
     def forward(self, inputs: List[torch.Tensor], **kwargs):
```

Comment:
nit: Since we're passing in max_new_tokens explicitly, let's add `max_new_tokens` as a parameter here.

Reply:
done